Doc/library/argparse.rst

```diff
@@ -1806,7 +1538,7 @@ Sub-commands
    Description of parameters:
 
    * title - title for the sub-parser group in help output; by default
-     "subcommands" if description is provided, otherwise uses title for
+     "sub-commands" if description is provided, otherwise uses title for
```

Comment:
The default is actually "subcommands" at runtime, not "sub-commands" (line 1826 of argparse.py at the moment). We should probably standardize on that spelling in the docs too.

Reply:
Ah, right. That makes sense. I'll err on the side of "subcommands"